website/source/docs/guides/deployment.html.md

```diff
@@ -3,18 +3,16 @@ layout: "docs"
 page_title: "Production Deployment"
 sidebar_current: "docs-guides-deployment"
 description: |-
-  Best practice approaches for Consul production deployments.
+  Best practice approaches for Consul production architecture.
 ---
 
-# Consul Production Deployment Guide
+# Consul Production Deployment
 
-As applications are migrated to dynamically provisioned infrastructure, scaling services and managing the communications between them becomes challenging. Consul’s service discovery capabilities provide the connectivity between dynamic applications. Consul also monitors the health of each node and its applications to ensure that only healthy service instances are discovered. Consul’s distributed runtime configuration store allows updates across global infrastructure.
-
-The goal of this document is to recommend best practice approaches for Consul production deployments. The guide provides recommendations on system requirements, datacenter design, networking, and performance optimizations for Consul cluster based on the latest Consul (1.0.7) release.
+The goal of this document is to recommend best practice approaches for Consul production deployments. The guide provides recommendations on system requirements, datacenter design, networking, and performance optimizations for Consul cluster based on the latest Consul (1.2.1) release.
```

Comment:
I recommend we don't include the version number - we are _bound_ to forget to update this on every release. "latest Consul release" seems perfectly clear.

Reply:
Fixed in ff690b68f